openupgradelib/openupgrade.py

```diff
@@ -1056,12 +1060,16 @@ def wrapped_function(cr, version):
                 logger.info(
                     "%s: %s-migration script called with version %s" %
                     (module, stage, version))
+                # Set up useful default context values
+                context = context or {}
+                context['migrate_version'] = version and version or None
+                context['module_name'] = module and module or None
```

Comment:
I propose you come up with a PR on OpenUpgrade where such a thing is used. For the time being, I strongly disagree with cluttering this lib with stuff that might be used some time somewhere.

This goes for most of your input up to now: I personally won't even have a look at huge change proposals that are not grounded in some already existing PR in OpenUpgrade.